runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java

```diff
@@ -144,7 +144,7 @@ public FlinkPipelineOptions getPipelineOptions() {
 
   @Override
   public String toString() {
-    return "DataflowRunner#" + hashCode();
+    return "FlinkRunner#" + hashCode();
```

Comment:
We could also include the provided PipelineOptions here. Doesn't have to be part of this PR though.